### PR TITLE
Write a blog post to announce Tuist 1.3.0

### PR DESCRIPTION
**Pull request by pepibumur**
As the title says, this PR adds a blog post to announce the latest version of Tuist. I took the opportunity to fix some minor nitpicks that I found in the documentation.

### CODE REVIEW

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+As the project grows,
+it's crucial for users, contributors, and maintainers to know what are our design pillars.
+For that reason,
+we updated our documentation to include a [manifesto](/docs/contribution/manifesto/) that contains a set of principles that are foundational to the design and development of Tuist.
+They help align users and contributors with the vision of the project.
+
+## Installation from Google Cloud Storage
+
+Tuist used to install new versions of Tuist by pulling the binaries from the GitHub releases.
+As soon as users reported, that caused some issues because GitHub's API returned limit errors.
```
Comment (fortmarek) [inline website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: `As soon as users reported, that caused...` Maybe rather something like: But we have got reports from users that this has caused ... ? The sentence, as it is written now, does not make sense, I think

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- Updated XcodeProj to 7.8.0
+- Added "Base" to the list of known regions - [Link](https://github.com/tuist/tuist/pull/1021)
+
+## What’s next
+
+Our goal is to provide teams with a full-fledged tool that helps them deal with their projects at scale. The list of challenges that teams face when scaling up their projects is long:
+
+- Keep consistency across projects.
+- Ensure that projects are simple and that can be modularized easily.
+- Integrate the management of certificates into.
```
Comment (fortmarek) [inline website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: I'd delete `into`

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Moreover, we are implementing caching by leveraging [xcframeworks](https://developer.apple.com/videos/play/wwdc2019/416/) and project generation. Teams might need to simplify their projects and remove side effects, for example, introduced by script build phases, but the benefits are huge. The only source code that developers will need to compile is the code of the module that they are working on. Transitive and non-transitive dependencies will be xcframeworks.
+
+> In other words, we are making scaling Xcode projects easy and fun for the rest of us
+
+It’s an ambitious plan but we are confident we can get there. We can’t move as fast as we’d like to but this slow yet with a steady pace is helping us keep an eye on the architecture and ensuring it evolves sustainably with the project. If you’d like to be part of the journey and contribute to making it happen join our [Slack](https://slack.tuist.io) and let us know. We’ll gladly onboard you and give you pointers to start contributing.
```
Comment (fortmarek) [inline website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: I'd delete `with a` (ie `this slow yet steady pace`)

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx (hunk 1 above)
Comment (pepibumur) [reply website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: You are right. It should be:

```
As users reported that caused some...
```

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx (hunk 2 above)
Comment (pepibumur) [reply website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: The sentence was actually incomplete. I rephrased it to:

```
Seamless management of certificates and profiles.
```

File: website/markdown/posts/2020-02-26-version-1.3.0/post.mdx (hunk 3 above)
Comment (pepibumur) [reply website/markdown/posts/2020-02-26-version-1.3.0/post.mdx]: Good idea. Changed ✅

**Pull request merged**